src/pyenphase/updaters/tariff.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from ..const import URL_TARIFF, SupportedFeatures
+from ..models.envoy import EnvoyData
+from ..models.tariff import EnvoyTariff
+from .base import EnvoyUpdater
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class EnvoyTariffUpdater(EnvoyUpdater):
+    """Class to handle updates for the Envoy tariff data."""
+
+    async def probe(
+        self, discovered_features: SupportedFeatures
+    ) -> SupportedFeatures | None:
+        return discovered_features
```

Comment:
```suggestion
        return None
```

Nothing new gets discovered here